app/controllers/sign_up/completions_controller.rb

```diff
@@ -36,9 +34,20 @@ def handle_verified_attributes
       clear_verify_attributes_sessions
     end
 
-    def show_completions_page?
-      service_providers = sp_session[:issuer].present? || @view_model.user_has_identities?
-      user_fully_authenticated? && service_providers
+    def redirect_to_account_if_no_issuer
+      redirect_to account_url if sp_session[:issuer].blank?
+    end
+
+    def redirect_to_sp_if_user_has_identity_for_issuer
```

Comment:
This may cause issues. We want to show the consent page if the attributes the SP is requesting have changed, even if the user has been to the SP before.

Reply:
Right but for IAL1 the attributes are always the same

Reply:
Ah I see I need to handle IAL2!